tests/jballerina-unit-test/src/test/resources/test-src/expressions/lambda/iterable/iterable-arrow-expression.bal

```diff
@@ -34,7 +34,7 @@ function testFilterIterable() returns float {
         index += 1;
         return value;
     }).filter(num => num[0] >= 3).map(num => num[1]);
-    return (<float>(ints:sum(...avg)))/<float>avg.length();
+    return (<float>(ints:sum(...avg))) / <float>avg.length();
```

Comment:
Is this a parser bug?

Reply:
Possibly a bug in the old parser (if not, an ambiguity in the spec). 
`a/<b>` matches to xml-step-expr

Reply:
Issue: 324099